.github/workflows/main.yml

```diff
@@ -40,30 +40,33 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven
 
-      - name: Maven operation with Sonar
-        if: matrix.sonar-enabled
+      - name: Regular build
         run: |
-          mvn -B -U -Pcoverage \
-          clean verify \
-          sonar:sonar \
+          mvn -B -U clean verify
```

Comment:
Maybe we should do good and move all the `mvn` to the wrapper `./mvnw`

Reply:
full agree